src/Measurements.jl

```diff
@@ -85,9 +83,17 @@ end
 # Start from 1, 0 is reserved to derived quantities
 const tag_counter = Threads.Atomic{UInt64}(1)
 
+
+if !isdefined(Base, :get_extension)
+    using Requires
+end 
```

Comment:
There's a trailing whitespace
```suggestion
end
```